Dockerfile

```diff
@@ -35,6 +37,9 @@ COPY package.json yarn.lock .nvmrc ./
 
 # Install the dependencies.
 RUN nvm install && nvm use
+RUN gem install wasmer
+RUN bundle package --all
+RUN bundle config set --local deployment 'true'
```

Comment:
This dockerfile is run in prod. Why do we need this line?

Reply:
This is needed to set deployment mode which will vendor the gems. The local flag is to tell bundler to check the local cache (where we installed wasmer) and use that. This gets around the bundler error we are currently seeing.